"Amino acid sequences, runs MPC or Super5 algorithm:\n"
"  muscle -mpc seqs.fa -out aln.fa\n"
"  muscle -super5 seqs.fa -out aln.fa\n"
"\n"
"Nucleotide or amino acid sequences, runs muscle v3 algorithm:\n"
"  muscle -in seqs.fa -out aln.fa [-maxiters 2]\n"
"\n"
"Input is FASTA, output is aligned FASTA.\n"
"\n"
"For amino acid sequences, use MPC for up to around 500 sequences, for larger\n"
"sets, use Super5. The muscle v3 algorithm is less accurate, not recommended.\n"
"\n"
"For nucleotide sequences, use -maxiters 2 if the default is too slow, which\n"
"typically happens with a few hundred sequences.\n"
"\n"
"mpc and super5:\n"
"  -perturb n           Integer seed for randomized HMM perturbations (default 0).\n"
"                         To make an ensemble, run several times with different seeds.\n"
"                         Zero (default) disables perturbing HMM parameters.\n"
"  -perm type           Guide tree permutation none, abc, acb, or bca (default none).\n"
"  -hmmin filename      Read HMM parameters from file.\n"
"  -hmmout filename     Save HMM parameters to file.\n"
"  -consiters n         Nr consistency iterations (default 2).\n"
"  -refineiters n       Nr refinement iterations (default 100).\n"
"  -threads n           Use n threads. Default is min(number of CPU cores, 20).\n"
"\n"
"super5:\n"
"  -maxcoarse n         Max size for coarse clusters (default 500).\n"
"  -paircount n         Max pairs for profile alignment (default 2000).\n"
"\n"
"muscle v3:\n"
"  -maxiters n          Max iterations. 2 is progressive, >2 does refinement.\n"
"                       Default is continue until convergence.\n"
"\n"
"All algorithms:\n"
"  -quiet               Turn off progress messages.\n"
"  -log filename        Write time, memory use and progress messages to file.\n"
